debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java

```diff
@@ -18,10 +18,7 @@
 import java.sql.SQLException;
 import java.time.Duration;
 import java.time.ZoneOffset;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
We shouldn't have `.*` imports, which I think the checkstyle job identified, too.